android/javatests/org/chromium/chrome/browser/BytecodeTest.java

```diff
@@ -804,6 +804,14 @@ public void testMethodsExist() throws Exception {
                         true,
                         int.class,
                         Context.class));
+        Assert.assertTrue(
+                methodExists(
+                        "org/chromium/chrome/browser/undo_tab_close_snackbar/UndoBarController",
+                        "showUndoBar",
+                        MethodModifier.REGULAR,
+                        true,
+                        List.class,
```

Comment:
seems like return type `void.class` is missing

Reply:
Oops! Added now.